core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -34,6 +34,9 @@ class SearchService(
     with FileChangeListener
     with SLF4JLogging {
 
+  private[indexer] def isUserFile(file: File): Boolean =
+    config.allRoots exists (r => file descendsFrom r)
```

Comment:
I think we need a test that fails on the current codebase to be sure that we're doing the right thing.